docs/user.md

```diff
@@ -10,25 +10,21 @@ Table Of Contents:
 
 - [Process](#process)
 - [Prerequisites](#prerequisites)
```

Comment:
Those tabulation changes were required to keep more compatibility with `mkdocs` rendering:
![Screenshot from 2022-10-26 19-34-41](https://user-images.githubusercontent.com/3216894/198151396-d621ba61-a516-47dd-873f-9724751a9079.png)

Reply:
PR created to publish the Github pages in mkdocs format: https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/24